core/nodejsActionBase/src/service.js

```diff
@@ -155,10 +174,12 @@ function NodeActionService(config) {
 
     function doRun(req) {
         var msg = req && req.body || {};
+        // Move per-activation keys to process env. vars with __OW_ (reserved) prefix)
```

Comment:
ending brace is not balanced.

Reply:
hey Rodric, I am not able to spot the imbalance.

Reply:
prefix) -> prefix